infra/app/service/notifications.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  # If this is a temporary environment, re-use an existing email identity. Otherwise, create a new one.
+  email_identity_arn = module.app_config.enable_notifications ? (
+    !local.is_temporary ? module.notifications[0].email_identity_arn : module.existing_notifications[0].email_identity_arn
+  ) : null
+  email_identity_config = module.app_config.enable_notifications ? (
+    !local.is_temporary ? module.notifications[0].email_identity_config : module.existing_notifications[0].email_identity_config
+  ) : null
+  notifications_environment_variables = module.app_config.enable_notifications ? {
+    PINPOINT_APP_ID = module.notifications_app[0].app_id,
+    PINPOINT_SENDER = local.notifications_config.sender_email
+  } : {}
```

Comment:
[via AWS's example code](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/python/example_code/pinpoint/pinpoint_send_email_message_api.py#L77-L80), these should be the only two things you need to use the Pinpoint API inside of an application